src/Questions/Checkbox/index.js

```diff
@@ -37,11 +37,11 @@ const QuestionCheckbox = ({
   const CustomComponent = ({ component }) => component(question, useForm)
 
   const MarkDownLink = ({ href, children }) =>
-    href === '#' ? (
+    href.startsWith('#') ? (
       <Link
         href={`${href}`}
         target='_self'
-        onClick={() => onLinkOpen(question.name)}
+        onClick={() => onLinkOpen(href.toString().substr(1))}
```

Comment:
Probably this need to be handle in a different way. What's the reason of the change? passing the question name should be enough to open and close a specific modal.